overlord/managers_test.go

```diff
@@ -6207,8 +6207,11 @@ func (ms *gadgetUpdatesSuite) SetUpTest(c *C) {
 	c.Assert(err, IsNil)
 }
 
-// makeMockDev puts a mock mount point under /run/mnt/{structureName}
-// that can then be used from mocked gadget.yaml
+// makeMockDev mocks /dev/disk/by-label/{structureName} and
+// /dev/disk/by-label/{structureName} under the test rootdir and for
```

Comment:
sorry, my suggestion was a bit messed up,   one of these should be /run/mnt/{structureName}